sklearn/linear_model/perceptron.py

```diff
@@ -73,6 +73,8 @@ class Perceptron(BaseSGDClassifier):
         When set to True, reuse the solution of the previous call to fit as
         initialization, otherwise, just erase the previous solution.
 
+        See :term:`the Glossary <warm_start>`.
```

Comment:
I would rather put the reference in the same paragraph (here and also some other places).

Reply:
Done